packages/bsky/src/data-plane/server/db/migrations/20240723T220703655Z-quotes.ts

```diff
@@ -0,0 +1,33 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('quote')
+    .addColumn('uri', 'varchar', (col) => col.primaryKey())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('subject', 'varchar', (col) => col.notNull())
+    .addColumn('subjectCid', 'varchar', (col) => col.notNull())
+    .addColumn('createdAt', 'varchar', (col) => col.notNull())
+    .addColumn('indexedAt', 'varchar', (col) => col.notNull())
+    .addColumn('sortAt', 'varchar', (col) =>
+      col
+        .generatedAlwaysAs(sql`least("createdAt", "indexedAt")`)
+        .stored()
+        .notNull(),
+    )
+    .execute()
+  await db.schema
+    .createIndex('quote_subject_idx')
+    .on('quote')
+    .column('subject')
+    .execute()
+  await db.schema
+    .createIndex('quote_order_by_idx')
+    .on('quote')
+    .columns(['sortAt', 'cid'])
+    .execute()
```

Comment:
I think this is the vibe!
```suggestion
  await db.schema
    .createIndex('quote_subject_cursor_idx')
    .on('quote')
    .columns(['subject', 'sortAt', 'cid'])
    .execute()
```